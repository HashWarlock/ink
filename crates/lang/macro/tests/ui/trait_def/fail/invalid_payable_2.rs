use ink_lang as ink;

#[ink::trait_definition]
pub trait InvalidPayable {
    #[ink(message, payable = true)]
    fn invalid_payable(&self);
}

fn main() {}
