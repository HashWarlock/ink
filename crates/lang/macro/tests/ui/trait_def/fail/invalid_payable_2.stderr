error: unknown ink! attribute argument (name = value)
 --> $DIR/invalid_payable_2.rs:5:20
  |
5 |     #[ink(message, payable = true)]
  |                    ^^^^^^^^^^^^^^
